lib/workarea/kount/mappings.rb

```diff
@@ -73,10 +73,10 @@ class Mappings
       }.freeze
 
       AUTO_MAPPING = {
-        'D' => :decline,
+        'D' => :declined,
```

Comment:
This is just to match the OOB implementation where `:declined` and `:approved` are the symbols we used.